5 - months with days/solution_5.rb

```diff
@@ -0,0 +1,27 @@
+require 'date'
+NEGATIVE_NUM = -1
+FIRST_MONTH = 1
+LAST_MONTH = 12
+
+class NewDate
+  def initialize(year)
+    @year = year
+    @months_with_days = generate_months_with_days_count
+  end
+
+  def generate_months_with_days_count
+    months_with_days = {}
+    (FIRST_MONTH..LAST_MONTH).each do |i|
+      months_with_days[Date::MONTHNAMES[i].to_s] = Date.new(@year, i, NEGATIVE_NUM).day
+    end
+    months_with_days
+  end
+
+  # select months of year with 30 days or any value days variable gets.
+  def get_months_with_days(days)
```

Comment:
please remove get_ from method names. I wrote about this in the chat
https://jtway.co/how-to-name-variables-and-methods-in-ruby-b4a21fb66c4d

Reply:
@cyberik-dev removed `get_`